Recruit-Training/Advanced-Recruit-Training/src/I_BatteryStateOfChargeService.h

```diff
@@ -6,10 +6,15 @@ struct BatteryData;
 class I_BatteryStateOfChargeService
 {
 public:
+    int hoursInteger;
+    int minutes;
+    int seconds;
+    double miliseconds;
+    double totalTimeHoursDouble;
```

Comment:
Errrr, I think there's some miscommunication here

I meant `totalTimeHoursDouble` didn't need to be private, since it's life span was only in `addData`

The other variables should still be private though